project/WORKSPACE

```diff
@@ -54,3 +54,76 @@ http_jar (
     name = "apache-commons-text",
     url = "https://repo1.maven.org/maven2/org/apache/commons/commons-text/1.9/commons-text-1.9.jar"
 )
+
+http_archive(
+    name = "com_google_j2cl",
+    strip_prefix = "j2cl-master",
```

Comment:
ditto, depending on `master` directly could be risky.

Reply:
Hm, I have failed to find any version [here](https://github.com/google/j2cl) which I can link to - there is only one release **v20190918**, but it's not full and doesn't contain `j2cl_maven_import_external` for Guava import 😶

Reply:
I see, never mind then, thanks for checking!